Assets/Scripts/Dialogue/DialogueManager.cs

```diff
@@ -259,25 +261,36 @@ public class DialogueManager : Manager<DialogueManager> {
             gameObject.SetActive(false);
             DialogueManager.Next(line);
         }
+		StartCoroutine (DelayClick ());
```

Comment:
The if/else statement preceding this is designed to skip dialogue lines that have no text. If a line is skipped, it should not trigger the click delay. (i.e. this line should be inside of the if statement).